<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singletest</class>
 <widget class="QDialog" name="singletest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>信号测试结果</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>信号频率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dsb_ns">
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-20000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>20000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>最大幅值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dsb_maxVolts">
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-20000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>20000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>最小幅值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dsb_minVolts">
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-20000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>20000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>是否根据信号最大峰值设置阈值？</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>保存并打开触发器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
